src/Chainweb/CutDB.hs

```diff
@@ -431,45 +458,49 @@ startCutDb config logfun headerStore payloadStore cutHashesStore = mask_ $ do
     -- 3. exitence of dependencies
     -- 4. full validation
     --
```

Comment:
I think, this whole comment is outdated now.

Reply:
It seems like maybe this comment was moved from somewhere else which makes more sense? Idk, I can't see why it's here.

Reply:
I deleted it under the presumption that it's outdated or inapplicable.